### PR TITLE
Optimisation changes for Mac page

### PR DESCRIPTION
**Pull request by ajeetraina**

### CODE REVIEW

File: docs/create/homebrew/index-homebrew.mdx [hunk 1]
```diff
@@ -1,11 +1,28 @@
 ---
 id: index-homebrew
-title: Create Redis database on Mac 
-sidebar_label: Redis on Mac
+title: Create Redis database on Mac OS
```
Comment (simonprickett) [inline docs/create/homebrew/index-homebrew.mdx]:
```suggestion
title: Create a Redis database on macOS 
```

File: docs/create/homebrew/index-homebrew.mdx [hunk 2]
```diff
@@ -1,11 +1,28 @@
 ---
 id: index-homebrew
-title: Create Redis database on Mac 
-sidebar_label: Redis on Mac
+title: Create Redis database on Mac OS 
+sidebar_label: Redis on Mac OS
```
Comment (simonprickett) [inline docs/create/homebrew/index-homebrew.mdx:4]:
```suggestion
sidebar_label: Redis on macOS
```

File: docs/create/homebrew/index-homebrew.mdx [hunk 3]
```diff
@@ -32,7 +49,7 @@ This will install all Redis and Redis Stack binaries. How you run these binaries
   🍺  redis-stack was successfully installed!
  ```
 
-:::info TIP
+:::info INFO
 
 If this is the first time you’ve installed Redis on your system, then all Redis Stack binaries will be installed and accessible from the `$PATH`. On M1 Macs, this assumes that `/opt/homebrew/bin` is in your path. On Intel-based Macs, /usr/local/bin should be in your path.
```
Comment (simonprickett) [inline docs/create/homebrew/index-homebrew.mdx]:
```suggestion
If this is the first time you’ve installed Redis on your system, then all Redis Stack binaries will be installed and accessible from the `$PATH`. On M1 Macs, this assumes that `/opt/homebrew/bin` is in your path. On Intel-based Macs, `/usr/local/bin` should be in your path.
```

File: docs/create/homebrew/index-homebrew.mdx [hunk 4]
```diff
@@ -131,6 +148,118 @@ RedisTimeSeries supports downsampling with the following aggregations: avg, sum,
 ![time series compaction](images/testredis6.png)
 
 
+</TabItem>
+<TabItem value="redis-on-macos">
+
+There are two ways to install Redis on Mac OS:
```
Comment (simonprickett) [inline docs/create/homebrew/index-homebrew.mdx:154]:
```suggestion
There are two ways to install Redis on macOS:
```

File: docs/create/homebrew/index-homebrew.mdx [hunk 5]
```diff
@@ -131,6 +148,118 @@ RedisTimeSeries supports downsampling with the following aggregations: avg, sum,
...
+
+- [Installing Redis from source](/create/from-source/)
+- Using Homebrew
+
+Homebrew is the easiest and most flexible way to install Redis on MacOS. It is a package management software for MacOS.
```
Comment (simonprickett) [inline docs/create/homebrew/index-homebrew.mdx]:
```suggestion
Homebrew is the easiest and most flexible way to install Redis on macOS. It is a package management software for macOS.
```

File: docs/create/homebrew/index-homebrew.mdx [hunk 6]
```diff
@@ -131,6 +148,118 @@ RedisTimeSeries supports downsampling with the following aggregations: avg, sum,
...
+It automates Redis installation process, making it quick and easy to add Redis to your system.
```
Comment (simonprickett) [inline docs/create/homebrew/index-homebrew.mdx]:
```suggestion
It automates the Redis installation process, making it quick and easy to add Redis to your system. 
```

File: docs/create/homebrew/index-homebrew.mdx [hunk 7]
```diff
@@ -131,6 +148,118 @@ RedisTimeSeries supports downsampling with the following aggregations: avg, sum,
...
+
+There are two ways to install Redis on Mac OS:
+
+- [Installing Redis from source](/create/from-source/)
+- Using Homebrew
+
+Homebrew is the easiest and most flexible way to install Redis on MacOS. It is a package management software for MacOS.
+It automates Redis installation process, making it quick and easy to add Redis to your system. 
+
+Follow the below steps to install Redis on Mac OS using brew service:
```
Comment (simonprickett) [inline docs/create/homebrew/index-homebrew.mdx]:
```suggestion
Follow the below steps to install Redis on macOS using the Homebrew package manager:
```

File: docs/create/homebrew/index-homebrew.mdx [hunk 8]
```diff
@@ -131,6 +148,118 @@ RedisTimeSeries supports downsampling with the following aggregations: avg, sum,
...
+
+
+### Step 1: Install Homebrew
+
+Run the below command to install and start brew service:
```
Comment (simonprickett) [inline docs/create/homebrew/index-homebrew.mdx]:
```suggestion
Run the following command to install and start brew service:
```

File: docs/create/homebrew/index-homebrew.mdx [hunk 9]
```diff
@@ -131,6 +148,118 @@ RedisTimeSeries supports downsampling with the following aggregations: avg, sum,
...
+
+
+ ```bash
+  /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
+ ```
+
+### Step 2: Install Redis using Homebrew package manager
+
+Follow the below command line to install Redis using brew service:
```
Comment (simonprickett) [inline docs/create/homebrew/index-homebrew.mdx]:
```suggestion
Use the following commands to install Redis using brew service:
```

File: docs/create/homebrew/index-homebrew.mdx [hunk 10]
```diff
@@ -131,6 +148,118 @@ RedisTimeSeries supports downsampling with the following aggregations: avg, sum,
...
+ ```bash
+  brew update
+  brew install redis
+ ```
+
+
+### Step 3: Start Redis server
+
+
+Run the below command line to start the Redis database in the background:
```
Comment (simonprickett) [inline docs/create/homebrew/index-homebrew.mdx]:
```suggestion
Run the following command to start the Redis database in the background:
```

File: docs/create/homebrew/index-homebrew.mdx [hunk 11]
```diff
@@ -131,6 +148,118 @@ RedisTimeSeries supports downsampling with the following aggregations: avg, sum,
...
+
+ ```bash
+  brew services start redis
+ ```
+
+In order to run the latest version of Redis, you will need to compil Redis from the source.
```
Comment (simonprickett) [inline docs/create/homebrew/index-homebrew.mdx]:
```suggestion
In order to run the latest version of Redis, you will need to compile Redis from the source.
```

File: docs/create/homebrew/index-homebrew.mdx [hunk 12]
```diff
@@ -131,6 +148,118 @@ RedisTimeSeries supports downsampling with the following aggregations: avg, sum,
...
+
+ ```bash
+  redis-cli
+  redis> set foo bar
+  OK
+  redis> get foo
+  "bar"
+ ``` 
+
+Please note: By default, Redis has 0-15 indexes for databases, you can change that number databases NUMBER in redis.conf.
```
Comment (simonprickett) [inline docs/create/homebrew/index-homebrew.mdx]: Suggest we remove this as using `SELECT` and the numbered databases is considered an antipattern and we shouldn't encourage people to use it.
```suggestion
```

File: docs/create/homebrew/index-homebrew.mdx [hunk 13]
```diff
@@ -131,6 +148,118 @@ RedisTimeSeries supports downsampling with the following aggregations: avg, sum,
...
+ ```
+
+### Step 9: Uninstall Redis 
+
+ ```bash
+  brew uninstall redis
+ ```
+
+
+### Next Step
```
Comment (simonprickett) [inline docs/create/homebrew/index-homebrew.mdx]:
```suggestion
### Next Steps
```

**Review by simonprickett — APPROVED** [review]
Approved with minor suggestions.

File: docs/create/homebrew/index-homebrew.mdx (hunk 1 above)
Comment (ajeetraina) [reply docs/create/homebrew/index-homebrew.mdx]: The space between Mac and OS has been added intentionally for SEO purpose.

File: docs/create/homebrew/index-homebrew.mdx (hunk 7 above)
Comment (ajeetraina) [reply docs/create/homebrew/index-homebrew.mdx]: The term "brew service" and "Mac OS" has been added for SEO purpose.

**Pull request merged**